tests/PKSim.Tests/IntegrationTests/DatabaseUpdateSpecs.cs

```diff
@@ -43,8 +102,7 @@ public void should_set_the_body_surface_area_parameter_of_an_individual_to_can_b
       [Observation]
       public void only_the_species_human_should_have_the_flag_is_human()
       {
-         var flatSpeciesRepository = IoC.Resolve<IFlatSpeciesRepository>();
```

Comment:
Why did you move those guys global? They are only required locally correct?

Reply:
just to have all IoC.Resolves in one place. not really required here, you're right

Reply:
Ok cool. I like it